From 83ad9a4489eb4a4689569e7d876f974374bea12c Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:37 -0400
Subject: [redhat] kabi: Adding symbol blk_execute_rq (block/blk-exec.c)

Message-id: <42a75f001367bfdab1610fefc12251623b0f5443.1410510773.git.poros@redhat.com>
Patchwork-id: 95925
O-Subject: [PATCH RHEL7.1 037/233] kabi: Adding symbol blk_execute_rq (block/blk-exec.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to insert a request into the lower
    level queue of the best path among all available paths to a lun.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_execute_rq
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_execute_rq
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_execute_rq

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_execute_rq b/redhat/kabi/kabi-module/kabi_ppc64/blk_execute_rq
new file mode 100644
index 0000000..67ba36b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_execute_rq
@@ -0,0 +1,2 @@
+#1-
+0x6edff580 blk_execute_rq vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_execute_rq b/redhat/kabi/kabi-module/kabi_s390x/blk_execute_rq
new file mode 100644
index 0000000..9273bde
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_execute_rq
@@ -0,0 +1,2 @@
+#1-
+0xe69a8bb7 blk_execute_rq vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_execute_rq b/redhat/kabi/kabi-module/kabi_x86_64/blk_execute_rq
new file mode 100644
index 0000000..7a0cc24
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_execute_rq
@@ -0,0 +1,2 @@
+#1-
+0x1fb99830 blk_execute_rq vmlinux EXPORT_SYMBOL
-- 
1.7.1